### PR TITLE
Malayalam Cheat Sheet

### PR DESCRIPTION
**Pull request by mohan08p**
What problem does your Instant Answer solve (Why is it better than organic links)?
It will help to find Basic malayalam words and phrases instantly for new people who don't know malayalam. It's a Dravidian language spoken in the southern Indian state of Kerala by about 35 million people. It is closely related to Tamil.

What is the data source for your Instant Answer? (Provide a link if possible)
http://www.pravasimalayalam.com/index.shtml

Why did you choose this data source?
It's good for learners and also everything is mentioned properly. So, we can go with the track and achieve the milestone while learning.

Are there any other alternative (better) data sources?
not that I know of.

What are some example queries that trigger this Instant Answer?
malayalam, malayalam phrases, malayalam phrases, english to malayalam, basic malayalam phrases, basic malayalam

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)
Communication is better if you are traveling to south india. Every person who want to know a little bit about Malayalam language.

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?
Yes, https://duck.co/ideas/idea/6760/malayalam-cheat-sheet

Which existing Instant Answers will this one supercede/overlap with?
Noop. This is new IA.

Are you having any problems? Do you need our help with anything?
No.

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)
malayalam-cheatsheet_1024

-----
IA Page: https://duck.co/ia/view/malayalam_cheat_sheet

### CODE REVIEW

**Comment by jophab** [comment]
@mohan08p  I think 'nanni' is better than 'nandi' for the word 'thankyou'.

 {
            "key" : "How do you say \"X\" in Bengali?",
            "val" : "നിങ്ങൾ  \"X\" ഇത് മലയാളത്തിൽ എങ്ങനെ പറയും ?",
            "trn" : "ningal  \"X\" eth malayalathil engane parayum?"
         },
Key value says "bengali"

**Comment by jophab** [comment]
{
            "key" : "You're welcome",
            "val" : "നിനക്ക് സ്വാഗതം ",
            "trn" : "ninak swagatham"
         }
lets change 'നിനക്ക്' to "താങ്കൾക്ക്" and "ninak" to "thaangalku" because its more polite

**Comment by jophab** [comment]
{
            "key" : "Can you speak more slowly, please?",
            "val" : "നിങ്ങള്ക്ക് ദയവായി വളരെ പതുകെ സംസാരിക്കാൻ കഴിയുമോ ?",
            "trn" : "ningalk dayavaayi valare pathuke samsarikan kazhiyumo?"
         },
lets change val to "താങ്കൾക്ക് കുറച്ചുകൂടി പതുക്കെ  സംസാരിക്കാൻ കഴിയുമോ? "
trn to "thangalku kurachukoodi pathukke samsarikkan kazhiyumo?"

**Comment by jophab** [comment]
{
            "key" : "Can you repeat, please?",
            "val" : "നിങ്ങൾ ദയവായി ആവര്തികുമോ ?",
            "trn" : "ningal dayavaayi aavarthikumo?"
         },
lets change "ആവര്തികുമോ" to "ആവർത്തിക്കുമോ "

**Comment by jophab** [comment]
{
            "key" : "Do you speak English?",
            "val" : "നീ ഇംഗ്ലീഷ് സംസാരിക്കുമോ ?",
            "trn" : "nee English samsarikumo?"
         },
change  നീ to താങ്കൾ and nee to thaangal

**Comment by jophab** [comment]
{
            "key" : "Can you write that down, please?",
            "val" : "നിങ്ങള്ക്ക്  അത് ദയവായി എഴുതാൻ കഴിയുമോ ?",
            "trn" : "Thangalk Ath ezhuthan pattumo?"
         },
 change  val to താങ്കൾക്ക്  അത് എഴുതാൻ കഴിയുമോ ?
trn to "thaangalkku ath ezhuthan kazhiyumo?"

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "Call an ambulance!",
+            "val" : "ഒരു ആംബുലൻസ് വിളിക്കു !",
+            "trn" : "oru ambulance viliku!"
+         },
+         {
+            "key" : "I am in danger!",
+            "val" : "ഞാൻ അപകടതിലാണ്ണ്‍  !",
+            "trn" : "njan apakadathil aan!"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: അപകടത്തിലാണ് is right
njan apakadathil aanu!

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "Turn left.",
+            "val" : "ഇടത്തോട്ട് തിരിയുക .",
+            "trn" : "edathott thiriyuka."
+         },
+         {
+            "key" : "Where is the exit?",
+            "val" : "എക്സിറ്റ് എവിടെയാണ് ?",
+            "trn" : "exut evideyaan?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: typo exit in trn
In trn change "evideyaan" to evideyaanu

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "Where?",
+            "val" : "എവിടെ ?",
+            "trn" : "evide?"
+         },
+         {
+            "key" : "Is it far?",
+            "val" : "അത് ബഹുദൂരം തന്നെയല്ലേ ?",
+            "trn" : "ath bahudooram thannayalle?"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: അത് ദൂരെ ആണോ ?
athu dhoore aano?
Its better

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "I am in danger!",
+            "val" : "ഞാൻ അപകടതിലാണ്ണ്‍  !",
+            "trn" : "njan apakadathil aan!"
+         },
+         {
+            "key" : "Can you help me?",
+            "val" : "എന്നെ സഹായിക്കാമോ ?",
+            "trn" : "enne sahayikamo?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: sahayikkamo

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+            "val" : "തീ !",
+            "trn" : "thee!"
+         }
+      ],
+      "Basics" : [
+         {
+            "key" : "My name is...",
+            "val" : "എന്റെ പേര്...",
+            "trn" : "ente per..."
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: peeru

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "I'm fine, thanks!",
+            "val" : "എനിക്ക് സുഖമാണ്, നന്ദി !",
+            "trn" : "enik sugamaan, nandi!"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: nandi to nanni

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "Welcome",
+            "val" : "സ്വാഗതം",
+            "trn" : "swagatham"
+         },
+         {
+            "key" : "What's your name?",
+            "val" : "എന്താണ് നിങ്ങളുടെ പേര് ?",
+            "trn" : "enthaan ningalude per?"
+         }
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: enthanu ningalude peeru?

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "Go straight ahead.",
+            "val" : "നേരെ മുന്നോട്ടു പോകുക .",
+            "trn" : "nere munnott pokuka."
+         },
+         {
+            "key" : "Do you have a city map?",
+            "val" : "നിങ്ങളുടെ കയ്യില പട്ടണത്തിന്റെ മാപ് ഉണ്ടോ ?",
+            "trn" : "ningalude kayyil pattanathinte map undo?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: കൈയ്യിൽ 
പട്ടണത്തിൻറെ
മാപ്പ്

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 9]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "Where can I find the taxis?",
+            "val" : "എനിക്ക് എവിടെ ടാക്സി കണ്ടെത്താൻ കഴിയും ?",
+            "trn" : "enik evide taxi kandethan kazhiyum?"
+         },
+         {
+            "key" : "How can I get a taxi?",
+            "val" : "എനിക്ക് എങ്ങനെ ടാക്സി ലഭിക്കും ?",
+            "trn" : "enik engane tazi labikum?"
+         }
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: typo in trn taxi
labhikkum

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
+      ],
+      "Going out for dinner" : [
+         {
+            "key" : "Do you have vegetarian dishes?",
+            "val" : "എവിടെ വെഗിടരിയൻ വിഭവങ്ങൾ ഉണ്ടോ ?",
+            "trn" : "Evide vegetarian vibavangal undo?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: ഇവിടെ 
വെജിറ്റേറിയൻ 
ivide
vibhavangal

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 11]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "I am allergic",
+            "val" : "എനിക്ക് അലെർഗി ഉണ്ട് ",
+            "trn" : "Enik alergy und"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: അലെർജി

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 12]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "The check please!",
+            "val" : "ചെക്ക്‌ നല്കു !",
+            "trn" : "check nalku!"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: do you mean cheque?

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 13]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "A cup of coffee please.",
+            "val" : "ദയവായി ഒരു കപ്പ്‌ കാപ്പി തരു .",
+            "trn" : "dhayavaayi oru cup kaappi tharu."
+         },
+         {
+            "key" : "Where is a good restaurant?",
+            "val" : "എവിടെ ആണ്‍ നല്ല ഭക്ഷണശാല ?",
+            "trn" : "Evide aan nalla bhakshanashaala?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: എവിടെ ആണ് നല്ല  ഭക്ഷണശാല ഉള്ളത് ?
evide annu nalla bhakshanashaala ullathu?

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 14]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "A glass of water please.",
+            "val" : "ഒരു ഗ്ലാസ്‌ വെള്ളം തരുമോ .",
+            "trn" : "oru glass vellam tharumo."
+         },
+         {
+            "key" : "A table for two please!",
+            "val" : "ദയവായി രണ്ട് ആള്കുള്ള ടേബിൾ വേണം !",
+            "trn" : "dayavaayi rand aalkulla table venam"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: ആളുകൾക്കുള്ള
aalukalkkulla

File: share/goodie/cheat_sheets/json/malayalam.json [hunk 15]
```diff
@@ -0,0 +1,238 @@
+{
+   "id" : "malayalam_cheat_sheet",
+   "name" : "Malayalam Cheat Sheet",
+   "metadata" : {
+      "sourceName" : "Pravasi Malayalam.com",
+      "sourceUrl" : "http://www.pravasimalayalam.com/index.shtml"
+   },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/malayalam.json]: I think .com is not necessary

File: share/goodie/cheat_sheets/json/malayalam.json [hunk 16]
```diff
@@ -0,0 +1,238 @@
...
+   "aliases": [
+       "malayalam", "malayalam phrases", "malayalam phrases", "english to malayalam",
+       "basic malayalam phrases", "basic malayalam"
+   ],
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/malayalam.json]: two "malayalam phrases" in aliases

File: share/goodie/cheat_sheets/json/malayalam.json [hunk 17]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "Call the police!",
+            "val" : "പൊലീസിനെ വിളിക്കുക !",
+            "trn" : "Polisine viliku!"
+         },
+         {
+            "key" : "Can you speak more slowly, please?",
+            "val" : "നിങ്ങള്ക്ക് ദയവായി വളരെ പതുകെ സംസാരിക്കാൻ കഴിയുമോ ?",
+            "trn" : "ningalk dayavaayi valare pathuke samsarikan kazhiyumo?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/malayalam.json]: lets change val to "താങ്കൾക്ക് കുറച്ചുകൂടി പതുക്കെ സംസാരിക്കാൻ കഴിയുമോ? "
trn to "thangalku kurachukoodi pathukke samsarikkan kazhiyumo?"

File: share/goodie/cheat_sheets/json/malayalam.json [hunk 18]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "Can you repeat, please?",
+            "val" : "നിങ്ങൾ ദയവായി ആവര്തികുമോ ?",
+            "trn" : "ningal dayavaayi aavarthikumo?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/malayalam.json]: lets change "ആവര്തികുമോ" to "ആവർത്തിക്കുമോ "

File: share/goodie/cheat_sheets/json/malayalam.json [hunk 19]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "How do you say \"X\" in Bengali?",
+            "val" : "നിങ്ങൾ  \"X\" ഇത് മലയാളത്തിൽ എങ്ങനെ പറയും ?",
+            "trn" : "ningal  \"X\" eth malayalathil engane parayum?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/malayalam.json]: change bengali to malayalam

File: share/goodie/cheat_sheets/json/malayalam.json [hunk 20]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "Stop!",
+            "val" : "നിർത്തുക !",
+            "trn" : "nirthuka!"
+         },
+         {
+            "key" : "Do you speak English?",
+            "val" : "നീ ഇംഗ്ലീഷ് സംസാരിക്കുമോ ?",
+            "trn" : "nee English samsarikumo?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/malayalam.json]: change നീ to താങ്കൾ and nee to thaangal

File: share/goodie/cheat_sheets/json/malayalam.json [hunk 21]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "I need a doctor!",
+            "val" : "എനിക്ക് ഒരു ഡോക്ടറെ ആവശ്യമുണ്ട്  !",
+            "trn" : "enik oru doctore aavashyamund!"
+         },
+         {
+            "key" : "Can you write that down, please?",
+            "val" : "നിങ്ങള്ക്ക്  അത് ദയവായി എഴുതാൻ കഴിയുമോ ?",
+            "trn" : "Thangalk Ath ezhuthan pattumo?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/malayalam.json]: change val to താങ്കൾക്ക് അത് എഴുതാൻ കഴിയുമോ ?
trn to "thaangalkku ath ezhuthan kazhiyumo?"

File: share/goodie/cheat_sheets/json/malayalam.json [hunk 22]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "Thank you very much!",
+            "val" : "വളരെ നന്ദി !",
+            "trn" : "valare nandi!"
+         },
+         {
+            "key" : "You're welcome",
+            "val" : "നിനക്ക് സ്വാഗതം ",
+            "trn" : "ninak swagatham"
+         }
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/malayalam.json]: lets change 'നിനക്ക്' to "താങ്കൾക്ക്" and "ninak" to "thaangalku" because its more polite

**Comment by tagawa** [comment]
Thank you for sending this cheat sheet @mohan08p (and to @jophab for the edit suggestions).
It looks like the two identical files have been included in the pull request so please could you delete one and leave one in the "language" directory? Then the git commands would be:
```
git checkout malayalam
git rm share/goodie/cheat_sheets/json/malayalam.json
git commit -m "Deleted extra file"
git push origin malayalam
```
I think that should fix this pull request automatically.

**Comment by mohan08p** [comment]
thanks so much. @jophab & @tagawa .

File: share/goodie/cheat_sheets/json/language/malayalam.json (hunk 8 above)
Comment (mohan08p) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: @jophab : But, it's meaning is different !

**Comment by tagawa** [comment]
Thank you for contributing again @mohan08p!

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 23]
```diff
@@ -0,0 +1,238 @@
...
+            "key" : "Thank you very much!",
+            "val" : "വളരെ നന്ദി !",
+            "trn" : "Valare nanni!"
+         },
+         {
+            "key" : "You're welcome",
+            "val" : "നതാങ്കൾക്ക് സ്വാഗതം ",
+            "trn" : "Thaangalku swagatham"
+         }
+      ],
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: val field is താങ്കൾക്ക് സ്വാഗതം

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 24]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "Call the police!",
+            "val" : "പൊലീസിനെ വിളിക്കുക !",
+            "trn" : "Polisine viliku!"
+         },
+         {
+            "key" : "Can you speak more slowly, please?",
+            "val" : "നതാങ്കൾക്ക് കുറച്ചുകൂടി പതുക്കെ സംസാരിക്കാൻ കഴിയുമോ ?",
+            "trn" : "Thangalku kurachukoodi pathukke samsarikkan kazhiyumo?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: remove the ന from begining...same as above

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 25]
```diff
@@ -0,0 +1,238 @@
...
+            "key" : "Excuse me!",
+            "val" : "എക്സ്ക്യൂസ് മീ !",
+            "trn" : "Excuse me!"
+         },
+         {
+            "key" : "I need a doctor!",
+            "val" : "എനിക്ക് ഒരു ഡോക്ടറെ ആവശ്യമുണ്ട്  !",
+            "trn" : "Enik oru doctore aavashyamund!"
+         },
+         {
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: lets change enik to enikku

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 26]
```diff
@@ -0,0 +1,238 @@
...
+            "key" : "Can you write that down, please?",
+            "val" : "താങ്കൾക്ക് അത് എഴുതാൻ കഴിയുമോ ?",
+            "trn" : "Thaangalkku ath ezhuthan kazhiyumo?"
+         },
+         {
+            "key" : "Call an ambulance!",
+            "val" : "ഒരു ആംബുലൻസ് വിളിക്കു !",
+            "trn" : "Oru ambulance viliku!"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: change viliku to vilikku

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 27]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "I am in danger!",
+            "val" : "ഞാൻ അപകടതിലാണ്ണ്‍  !",
+            "trn" : "Njan apakadathil aanu!"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: അപകടത്തിലാണ് is right
njan apakadathil aanu!

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 28]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "How are you?",
+            "val" : "സുഖമാണോ ?",
+            "trn" : "Sugamaano?"
+         },
+         {
+            "key" : "Welcome",
+            "val" : "സ്വാഗതം",
+            "trn" : "Swagatham"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: swaagatham

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 29]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "Go straight ahead.",
+            "val" : "നേരെ മുന്നോട്ടു പോകുക .",
+            "trn" : "Nere munnott pokuka."
+         },
+         {
+            "key" : "Do you have a city map?",
+            "val" : "നിങ്ങളുടെ കയ്യില പട്ടണത്തിന്റെ മാപ് ഉണ്ടോ ?",
+            "trn" : "Ningalude kayyil pattanathinte map undo?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: change കയ്യില to കൈയ്യിൽ 
പട്ടണത്തിന്റെ to പട്ടണത്തിൻറെ
മാപ് to  മാപ്പ്

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 30]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "Turn right.",
+            "val" : "വലത്തോട്ട് തിരിയുക .",
+            "trn" : "Valathottu thiriyuka."
+         },
+         {
+            "key" : "Where can I find the taxis?",
+            "val" : "എനിക്ക് എവിടെ ടാക്സി കണ്ടെത്താൻ കഴിയും ?",
+            "trn" : "Enik evide taxi kandethan kazhiyum?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: typo in key 'taxis'

File: share/goodie/cheat_sheets/json/language/malayalam.json (hunk 8 above)
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: @mohan08p i am sorry that i dont explain what to do...
check my new comment for  the same portion

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 31]
```diff
@@ -0,0 +1,238 @@
...
+            "val" : "എനിക്ക് എങ്ങനെ ടാക്സി ലഭിക്കും ?",
+            "trn" : "Enik engane taxi labhikkum?"
+         }
+      ],
+      "Going out for dinner" : [
+         {
+            "key" : "Do you have vegetarian dishes?",
+            "val" : "എവിടെ വെഗിടരിയൻ വിഭവങ്ങൾ ഉണ്ടോ ?",
+            "trn" : "Ivide vegetarian vibhavangal undo?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: change എവിടെ to ഇവിടെ 
വെഗിടരിയൻ to  വെജിറ്റേറിയൻ

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 32]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "I am allergic",
+            "val" : "എനിക്ക് അലെർഗി ഉണ്ട് ",
+            "trn" : "Enik alergy und"
+         },
+         {
+            "key" : "The cheque please!",
+            "val" : "പരിശോധിക്കുക !",
+            "trn" : "Cheque nalku!"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: change val  to ചെക്ക്‌ നല്കു !

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 33]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "A cup of coffee please.",
+            "val" : "ദയവായി ഒരു കപ്പ്‌ കാപ്പി തരു .",
+            "trn" : "Dhayavaayi oru cup kaappi tharu."
+         },
+         {
+            "key" : "Where is a good restaurant?",
+            "val" : "എവിടെ ആണ്‍ നല്ല ഭക്ഷണശാല ?",
+            "trn" : "Evide annu nalla bhakshanashaala ullathu?"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: എവിടെ ആണ് നല്ല ഭക്ഷണശാല ഉള്ളത് ?
evide aanu nalla bhakshanashaala ullathu?

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 34]
```diff
@@ -0,0 +1,238 @@
...
+            "key" : "A glass of water please.",
+            "val" : "ഒരു ഗ്ലാസ്‌ വെള്ളം തരുമോ .",
+            "trn" : "Oru glass vellam tharumo."
+         },
+         {
+            "key" : "A table for two please!",
+            "val" : "ദയവായി രണ്ട് ആള്കുള്ള ടേബിൾ വേണം !",
+            "trn" : "Dayavaayi rand aalukalkkulla table venam"
+         },
+         {
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: change ആള്കുള്ള to  ആളുകൾക്കുള്ള

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 35]
```diff
@@ -0,0 +1,238 @@
...
+            "val" : " താങ്കൾക്ക് സ്വാഗതം ",
+            "trn" : "Thaangalku swaagatham"
+         }
+      ],
+      "Getting Help" : [
+         {
+            "key" : "Call the police!",
+            "val" : "പൊലീസിനെ വിളിക്കുക !",
+            "trn" : "Polisine viliku!"
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: change val to
 പൊലീസിനെ വിളിക്കൂ
current val field is right but the  trn given is for the above val.
so either change the val field or change the trn field to "polisine vilikkuka"
I recommend changing the val field .bcoz it is  the common usage while speaking.

Comment (mohan08p) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: @jophab : Thanks for all the proper corrections.

Comment (jophab) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: @mohan08p please make the correction for alergy too
look the above comments

**Comment by jophab** [comment]
@mohan08p please make the correction for alergy too
look the above comments

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 36]
```diff
@@ -0,0 +1,238 @@
...
+   ],
+   "template_type": "language",
+   "section_order" : [
+      "Etiquette",
+      "Basics",
+      "Getting Help",
+      "Travelling",
+      "Going out for dinner"
+   ],
+   "description" : "Basic Malayalam words and phrases",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: nit: move this to the top, below the `name`

File: share/goodie/cheat_sheets/json/language/malayalam.json [hunk 37]
```diff
@@ -0,0 +1,238 @@
+{
+   "id" : "malayalam_cheat_sheet",
+   "name" : "Malayalam Cheat Sheet",
+   "metadata" : {
+      "sourceName" : "Pravasi Malayalam",
+      "sourceUrl" : "http://www.pravasimalayalam.com/index.shtml"
+   },
+   "aliases": [
+       "malayalam", "malayalam phrases", "Malayalam phrases", "english to malayalam",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: `malayalam` isn't needed, it's already the default trigger because of the filename

Comment (moollaza) [inline share/goodie/cheat_sheets/json/language/malayalam.json]: `Malayalam phrases` -> `malayalam phrases` (no upper-case triggers please)

**Comment by moollaza** [comment]
I've set this up for deploy onto Beta. Once that's done I think this one will be ready for merge :+1:

**Comment by mohan08p** [comment]
Thanks. :) @moollaza.

**Comment by moollaza** [comment]
@mohan08p Thanks! and sorry for the delay.  This LGTM :+1:

**Comment by gautamkrishnar** [comment]
@mohan08p :+1:  Pwolichuttooo.... :dancer: :smile:

**Comment by jophab** [comment]
@gautamkrishnar he is from mumbai.
@mohan08p congrats :+1:

**Comment by gautamkrishnar** [comment]
@mohan08p Oh man! Great job! I thought that you were malayali :smile:

**Comment by mohan08p** [comment]
Thanks @moollaza , @jophab , @tagawa , @gautamkrishnar : This would not be possible without great community support !

**Comment by tagawa** [comment]
Good job @mohan08p!

**Comment by moollaza** [comment]
@mohan08p congrats! It's live: https://duckduckgo.com/?q=Malayalam+Cheat+Sheet&t=ffcm&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**